pkg/ansible/extra_vars.go

```diff
@@ -0,0 +1,27 @@
+package ansible
+
+import (
+	random "math/rand"
+	"time"
+
+	"github.com/mesosphere/konvoy-image-builder/pkg/constants"
+)
+
+type ExtraVars struct {
```

Comment:
Do we need this, these are not being used anywehre.

Reply:
yep, that is part of the clean up.

Reply:
actually it used as part of startPlaybook